lib/IRGen/GenDecl.cpp

```diff
@@ -1078,9 +1078,6 @@ void IRGenerator::emitGlobalTopLevel(llvm::StringSet<> *linkerDirectives) {
   // Emit differentiability witnesses.
   for (auto &dw :
            PrimaryIGM->getSILModule().getDifferentiabilityWitnessList()) {
-    if (dw.isDeclaration())
```

Comment:
Note: `if (dw.isDeclaration()) continue` is redundant because `IRGenModule::emitSILDifferentiabilityWitness` has the same logic.